spec/datadog/ci/ext/environment_spec.rb

```diff
@@ -24,5 +64,57 @@
         end
       end
     end
+
+    context 'inside a git repository' do
+      shared_context 'matching git tags' do |git_fixture, expected_tags|
+        let(:environment_variables) { { 'GIT_DIR' => "#{fixture_dir}/git/#{git_fixture}" } }
+
+        context "for #{git_fixture}" do
+          it 'matches tags' do
+            is_expected.to eq(expected_tags)
+          end
+        end
+      end
+
+      it_behaves_like 'matching git tags', 'gitdir_empty', {
+        'ci.workspace_path' => "#{Dir.pwd}/spec/datadog/ci/ext/fixtures/git",
+        'git.branch' => 'HEAD',
+        'git.commit.sha' => 'HEAD'
+      }
+
+      it_behaves_like 'matching git tags', 'gitdir_with_commit', {
+        'ci.workspace_path' => "#{Dir.pwd}/spec/datadog/ci/ext/fixtures/git",
+        'git.branch' => 'master',
+        'git.commit.author.date' => '2011-02-16T13:00:00+00:00',
```

Comment:
@juan-fernandez Is this date format what we expect for the CI app?

Reply:
yeah I think this should work. The backend is pretty flexible on this I think. I'm sending this format `Fri Jun 18 16:14:22 2021 +0200` and it works too. Easiest way would be to send some data to CI App from this PR: is that possible?